packages/react/.storybook/main.ts

```diff
@@ -1,6 +1,6 @@
 import { dirname, join } from "path";
 import { StorybookConfig } from "@storybook/react-vite";
-import remarkGfm from 'remark-gfm';
+import remarkGfm from "remark-gfm";
```

Comment:
Ditto line 14. I think you should include `.storybook` folder into the scope of eslint.